src/slider/tp-slider.ts

```diff
@@ -430,6 +449,9 @@ export class TPSliderElement extends HTMLElement {
 	 * @protected
 	 */
 	handleResize(): void {
+		// Update responsive settings.
+		this.updateResponsiveSettings();
```

Comment:
@imranhsayed This will be heavy for INP on a resize event, so lets do:

`setTimeout( () => { this.updateResponsiveSettings() }, 0 )`

Lets also add a comment that it is done for INP